manual/src/refman/extensions/extensionsyntax.etex

```diff
@@ -15,7 +15,7 @@ infix-symbol:
 ;
 prefix-symbol:
           ...
-        | ('?'||'~'||'!') { operator-char } "#" { operator-char || "#"}
+        | ('?' || '~' || '!') {operator-char} "#" {operator-char || "#"}
```

Comment:
Most of the use of `{...}` in the language part of the manual seems to add space `{ ... }` as far as I can see?